MCprep_addon/materials/generate.py

```diff
@@ -216,6 +217,32 @@ def find_from_texturepack(blockname: str, resource_folder: Optional[Path]=None)
 	return res
 
 
+def get_format_version_texturepack(resource_folder: Optional[Path]=None) -> Union[int, MCprepError]:
+	""" Get texturepack format version
+
+	See https://minecraft.wiki/w/Pack_format#List_of_resource_pack_formats"""
+	if resource_folder is None:
+		# default to internal pack
+		resource_folder = Path(cast(
+			str,
+			bpy.path.abspath(bpy.context.scene.mcprep_texturepack_path)
+		))
+
+	if not resource_folder.exists() or not resource_folder.is_dir():
+		env.log("Error, resource folder does not exist")
+		line, file = env.current_line_and_file()
+		return MCprepError(FileNotFoundError(), line, file, f"Resource pack folder at {resource_folder} does not exist!")
+
+	# Resource folder is same level as assets folder
+	file = Path(resource_folder, "pack.mcmeta")
+	if (file.is_file()):
```

Comment:
Extra parenthesis unnecessary, `if file.is_file()` should be sufficient. 

Also suggest extending this to `if file.exists() and file.is_file()` for consistency reasons